src/main/java/org/elasticsearch/action/support/nodes/NodesOperationRequest.java

```diff
@@ -43,6 +43,11 @@ protected NodesOperationRequest() {
 
     }
 
+    protected NodesOperationRequest(ActionRequest request, String... nodesIds) {
```

Comment:
all of the other requests that extend this one are user facing requests, this is the only case where we extended for an internal request that actually needed the headers copied from the parent request. We are good ;)